benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -577,6 +585,8 @@ function (swift_benchmark_compile_archopts)
       "-whole-module-optimization"
       "-emit-module" "-module-name" "${module_name}"
       "-I" "${objdir}"
+      "-Xfrontend" "-enable-cxx-interop"
+      "-I" "${srcdir}/utils/CxxTests/"
```

Comment:
Why does the main module need this?

Reply:
Because otherwise when compiling `main.swift` the compiler complains about missing the C++ includes:
```
/Users/zoe/Developer/swift-source/swift/benchmark/utils/main.swift:51:8: error: missing required module 'CxxTypes'
import CreateObjects
```